src/main/java/frc/robot/commands/AutoBalanceClimb.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ClimbingSubsystem;
+
+public class AutoBalanceClimb extends Command{
+    ClimbingSubsystem s_climbingSubsystem;
+    double leftSpeed;
+    double rightSpeed;
+
+    public AutoBalanceClimb(){
+        s_climbingSubsystem = ClimbingSubsystem.getInstance();
+        addRequirements(s_climbingSubsystem);
+    }
+
+    @Override
+    public void execute(){
+        if(s_climbingSubsystem.s_Swerve.getGyroRoll() < -5){
+            // left climber is going go negative in speed, right will be positive
+            leftSpeed = (s_climbingSubsystem.s_Swerve.getGyroRoll()) / 100;
+            rightSpeed = -(s_climbingSubsystem.s_Swerve.getGyroRoll()) / 100;
+        } else if (s_climbingSubsystem.s_Swerve.getGyroRoll() > 5){
+            leftSpeed = -(s_climbingSubsystem.s_Swerve.getGyroRoll()) / 100;
+            rightSpeed = (s_climbingSubsystem.s_Swerve.getGyroRoll()) / 100;
+        } else {
+            leftSpeed = 0;
+            rightSpeed = 0;
```

Comment:
We should change these speeds to whatever the max speed for those motors should be.